### PR TITLE
Adding functions to clean up code

### PR DESCRIPTION
**Pull request by rafgeekee**

### CODE REVIEW

File: calculator/main.js [hunk 1]
```diff
@@ -1,16 +1,28 @@
 var keys = document.querySelectorAll('#calculator span');
 var operators = ['+', '-', 'x', '÷'];
 var decimalAdded = false;
+var input = document.querySelector('.screen');
```
Comment (chrisprior1995) [inline calculator/main.js:4]: What is querySelector

Comment (rafgeekee) [reply calculator/main.js:4]: its like `getElementByID` But I'm selecting a class

Comment (noisyscanner) [reply calculator/main.js:4]: @chrisprior1995 vanilla JS equiv. of $('.screen')

Comment (RobertBMerriman) [inline calculator/main.js:4]: Improvement: querying for '.screen' only once instead of on every button click

File: calculator/main.js [hunk 2]
```diff
@@ -1,16 +1,28 @@
...
+}
+
+function clearScreen() {
+	input.innerHTML = '';
+	decimalAdded = false;
+}
 
 for(var i = 0; i < keys.length; i++) {
 	keys[i].onclick = function(e) {
-		var input = document.querySelector('.screen');
```
Comment (sonicintrusion) [inline calculator/main.js:7]: good job puling this out of the loop

File: calculator/main.js [hunk 3]
```diff
@@ -1,16 +1,28 @@
...
-		var inputVal = input.innerHTML;
+		var inputVal = getInput();
```
Comment (sonicintrusion) [inline calculator/main.js:21]: good job making use of the functions

**Review by CGDEV998 — APPROVED** [review]

**Review by RobertBMerriman — APPROVED** [review]

File: calculator/main.js (hunk 1 above)
Comment (RobertBMerriman) [reply calculator/main.js:4]: S'very nice

**Review by beccahalford — APPROVED** [review]

**Review by william-eyre — APPROVED** [review]
Yeah looks great

**Review by rbnbradford — APPROVED** [review]
V. nice much tidier

**Review by pas-github — APPROVED** [review]

**Pull request merged**